widget/ui/src/containers/SwapInput/SwapInput.types.ts

```diff
@@ -22,14 +22,14 @@ export type BaseProps = {
 };
 
 type FromProps = {
-  label: 'From';
+  label: string;
```

Comment:
I think it has some side effects on SwapInput implementation because of conditions on label.

Reply:
Done